src/UI/Components/Diagram/actions.ts

```diff
@@ -358,25 +358,27 @@ export function appendEmbeddedEntity(
     instanceAsTable.addTo(graph);
 
     //iterate through embedded entities to create and connect them
-    embeddedEntity.embedded_entities.forEach((entity) => {
-      const appendedEntity = appendEmbeddedEntity(
-        paper,
-        graph,
-        entity,
-        entityInstance[entity.name] as InstanceAttributeModel,
-        instanceAsTable.id as string,
-        embeddedEntity.name,
-        presentedAttr,
-        isBlockedFromEditing,
-      );
+    embeddedEntity.embedded_entities
```

Comment:
Can you please add to the comments in the code the reason why we are filtering these out.

Reply:
added

Reply:
de-cluttering is good but doesn't tell why we are doing this, I vaguely know it's about the alocated data, but I'd like to see an actual explanation on it why they don't need to be displayed.